src/utils/PermissionUtils.js

```diff
@@ -13,4 +13,12 @@ module.exports = class PermissionUtils {
       return member.roles.filter(r => r.hoist).sort((a, b) => b.position - a.position).first()
     }
   }
+
+  static async isDj (client, user, channel) {
+    const guild = channel.guild
+    const role = await client.database.guilds.get(guild)
+    if (!role) return
+    const hasRole = role && role.members.has(user.id)
+    return hasRole
```

Comment:
wait, this whole thing is verifying on a guildDocument if there's a user in it..?

Reply:
se tiver um cargo na database qualquer, ele vai procurar pelo cargo, se não tiver, ele vai deixar passar

Reply:
but bro, you're using ``role``, not ``{ role }`` or ``{ djRole }``

Reply:
you're defining a variable to the guildDocument, and not to the value on that certain guildDocument

Reply:
O